### PR TITLE
Add some documentation features

### PR DESCRIPTION
**Pull request by Matt-Gleich**
Signed-off-by: Matthew Gleich <email@mattglei.ch>

## Description

Added the following:

1. A fancy header to the main README.
2. A table of contents to the main README and the main CONTRIBUTING doc.
3. Refactored all markdown links to link to local files instead of the repo.

## Type of change

- [ ] New script command
- [ ] Bug fix
- [ ] Improvement of an existing script
- [x] Documentation update
- [ ] Toolkit change
- [ ] Other (Specify)

## Screenshot

Ohhh so fancy

Screen Shot 2021-01-17 at 5 15 29 PM

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Review by PitNikola — CHANGES_REQUESTED** [review]
Hey @Matt-Gleich, thanks so much for making this suggestion and sorry it took a while to review it. Generally looks great, I just have few suggestions:
- Since GitHub now support dark mode, using light variant of our logo might not look good there. Let's use our current app icon since it's theme agnostic. Here's png at 1024px. Feel free to resize to smaller size. I would also recommend making the logo a bit smaller.
raycast-logo-1024

- Let's remove first separator under the title, we think it would look cleaner.

Let me know what you think!

File: README.md [hunk 1]
```diff
@@ -1,16 +1,53 @@
...
+  - [Passing Arguments](#passing-arguments)
+  - [Error Handling](#error-handling)
+  - [Login Shell and `$PATH`](#login-shell-and-path)
+- [Troubleshooting](#troubleshooting)
+- [Community](#community)
+
 ## Awesome Script Commands
 
-You can find a lot of [useful script commands built by our community](https://github.com/raycast/script-commands/tree/master/commands) here.
+You can find a lot of [useful script commands built by our community](commands/) here.
```
Comment (unnamedd) [inline README.md:48]:
```suggestion
You can find a lot of [useful Script Commands built by our community](commands/) here.
```

File: README.md [hunk 2]
```diff
@@ -21,23 +58,23 @@ To install new commands, follow these steps:
 3. Click `Add More Directories`
 4. Select the Script Command that you want to install
 
-![Preferences](https://github.com/raycast/script-commands/blob/master/screenshots/preferences.png?raw=true)
+![Preferences](images/screenshots/preferences.png)
 
-To get started, download one of the sample commands in this repository or [write a custom script](https://github.com/raycast/script-commands/blob/master/README.md#write-script-commands).
+To get started, download one of the sample commands in this repository or [write a custom script](#write-script-commands).
```
Comment (unnamedd) [inline README.md:63]:
```suggestion
To get started, download one of the sample commands in this repository or [write a custom Script Command](#write-script-commands).
```

File: README.md [hunk 3]
```diff
@@ -21,23 +58,23 @@ To install new commands, follow these steps:
...
 
-Grab scripts that you want to use and copy them to a separate directory on your machine (you can use the [_enabled-commands](https://github.com/raycast/script-commands/blob/master/_enabled-commands) folder from this repo for this).
+Grab scripts that you want to use and copy them to a separate directory on your machine (you can use the [_enabled-commands](_enabled-commands/) folder from this repo for this).
 
 **Note**: We recommend against directly using script directories from this repo in Raycast to protect yourself from potential restructuring and new script commands suddenly appearing in Raycast.
 
 ## Write Script Commands
 
-To write your custom Script Commands, go over the following steps and follow the [troubleshooting section](https://github.com/raycast/script-commands/blob/master/README.md#troubleshooting) if your script command does not show up in Raycast.
+To write your custom Script Commands, go over the following steps and follow the [troubleshooting section](#troubleshooting) if your script command does not show up in Raycast.
```
Comment (unnamedd) [inline README.md:73]:
```suggestion
To write your custom Script Commands, go over the following steps and follow the [troubleshooting section](#troubleshooting) if your Script Command does not show up in Raycast.
```

File: README.md [hunk 4]
```diff
@@ -142,7 +179,7 @@ If a script doesn't appear in the commands list, make sure these requirements ar
 * You use either `#` or `//` comments for metadata parameters
 * You have reloaded the scripts, via 1) the "Reload" button in preferences or 2), the "Reload Script Directories" command in root search (v0.33+), or 3) automatically via the activated "Auto Reload" feature in preferences (v0.33+)
 
-If nothing helps, try to go step by step from a [template](https://github.com/raycast/script-commands/blob/master/script-command.template.sh) script command or use one of the examples in this repo.
+If nothing helps, try to go step by step from a [template](templates/script-command.template.sh) script command or use one of the examples in this repo.
```
Comment (unnamedd) [inline README.md:182]:
```suggestion
If nothing helps, try to go step by step from a [template](templates/script-command.template.sh) Script Command or use one of the examples in this repo.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -2,11 +2,23 @@
...
+    - [Bash profiles and environmental variables](#bash-profiles-and-environmental-variables)
+    - [Auto generated files](#auto-generated-files)
+  - [Examples](#examples)
+
 ## Guidelines
 
 ### Git and Pull Requests
 
-Please make sure that your pull request makes it easy for the reviewer to understand what the change is about (fill in the pull request template) and keep your change focused (do not create a PR containing multiple complex script commands). 
+Please make sure that your pull request makes it easy for the reviewer to understand what the change is about (fill in the pull request template) and keep your change focused (do not create a PR containing multiple complex script commands).
```
Comment (unnamedd) [inline CONTRIBUTING.md:21]:
```suggestion
Please make sure that your pull request makes it easy for the reviewer to understand what the change is about (fill in the pull request template) and keep your change focused (do not create a PR containing multiple complex Script Commands).
```

**Review by unnamedd — CHANGES_REQUESTED** [review]
From my perspective, the PR looks good, just need some adjusts in terms of cases.

**Comment by Matt-Gleich** [comment]
This is what the image looks like now:

Screen Shot 2021-01-31 at 5 25 07 PM

**Review by unnamedd — APPROVED** [review]
Looks good to me! Thanks for the changes @Matt-Gleich!

**Comment by PitNikola** [comment]
Awesome, thanks @Matt-Gleich!

**Pull request merged**